### DISCUSSION TITLE
How exactly can we activate maintenance mode by admin panel?

### DISCUSSION THREAD

**Discussion by cnr91** [fabiocaccamo/django-maintenance-mode#100] [General]
How exactly can we activate maintenance mode by admin panel? 

I published my project with amazon elastic beanstalk and this process by settings.py will not be very useful for me. 

```
#if True the maintenance-mode will be activated
MAINTENANCE_MODE = True
```

Thank you.

**Comment by fabiocaccamo** [comment]
You can do it in different ways:

- Using Terminal: https://github.com/fabiocaccamo/django-maintenance-mode#terminal

- Using URLs: https://github.com/fabiocaccamo/django-maintenance-mode#urls-1

- Using Python: https://github.com/fabiocaccamo/django-maintenance-mode#python

- If you want to control the maintenance mode state using a model admin, you can create your own backend which reads/writes the state from/to a model field (you just need to override the `get_value` and `set_value` methods and set `MAINTENANCE_MODE_STATE_BACKEND` setting to the backend path, eg. `yourapp.backends.MaintenanceModeAdminBackend`.

- Recently @paduszyk opened an [issue](https://github.com/fabiocaccamo/django-maintenance-mode/issues/95) about adding a maintenance mode toggle control in the admin, he was very kind and shared his solution, you can take inspiration from there if that is the solution you need.

**Reply by cnr91** [reply]
Thank you very much for the explanatory information.
